src/components/Navigation/Pages/PageBrowse/PageBrowse.js

```diff
@@ -15,15 +16,19 @@ import FilterBar from '../../Sections/FilterBar'
 import FullDetailPage from '../../../FullDetailView/FullDetailPage'
 import ProviderButtons from '../../Ui/ProviderButtons'
```

Comment:
these provider buttons are mixing types and provider.

```jsx
    <ButtonGroup>
        <Button name="github" onClick={onClick} active={activeProvider === 'github'}>
          GitHub
        </Button>
        <Button name="maven" onClick={onClick} active={activeProvider === 'maven'}>
          Maven
        </Button>
        <Button name="npm" onClick={onClick} active={activeProvider === 'npm'}>
          NPM
        </Button>
        <Button name="nuget" onClick={onClick} active={activeProvider === 'nuget'}>
          NuGet
        </Button>
        <Button name="crate" onClick={onClick} active={activeProvider === 'crate'}>
          Crate
        </Button>
        <Button name="pypi" onClick={onClick} active={activeProvider === 'pypi'}>
          PyPi
        </Button>
        <Button name="rubygems" onClick={onClick} active={activeProvider === 'rubygems'}>
          RubyGems
        </Button>
      </ButtonGroup>
```

the providers are:
 - cratesio
 - rubygems
 - github
 - mavencentral
 - npmjs
 - nuget
 - cocoapods
 - pypi

the types are:
 - crate
 - gem
 - git
 - maven
 - npm
 - nuget
 - pod
 - pypi
 - sourcearchive

I think the best case would actually be to align the providerButtons with the actual list of providers here and then change the query from type to provider.
Make sense?

Reply:
These buttons are also used in the Harvest section, i need to check if this change would have some impact to that section